rmgpy/cantherm/qchem.py

```diff
@@ -187,15 +190,13 @@ def loadGeometry(self):
     
     def loadConformer(self, symmetry=None, spinMultiplicity=0, opticalIsomers=1, symfromlog=None, label=''):
```

Comment:
I tried running two tests on some QChem files I had to see if Cantherm had the expected behavior:

1) Specify a symmetry number in the Cantherm input file and look at the symmetry number listed in the output.

This test was successful. As I changed the symmetry number in the Cantherm input file, the number in the output file changed as well, even though the QChem log file did specify a value.



2) Don't specify a symmetry number in the Cantherm input file and see if the QChem value is shown in the output.

This test failed. No matter what I set the number in the log to, Cantherm had an output of 1. I believe this is because in our code `symmetry` being `None` does not set the value of `symfromlog` to `True`. The other part of the problem is that the QChem 'Rotational Symmetry Number' occurs after the 'Eigenvalues --' line (seems to be true for QChem 4.4 as well as the latest version on NERSC), so `symmetry` is still `None` when lines 250-251 set the symmetry to a value of 1. When Cantherm finally does read in the symmetry number off of the 'Rotational Symmetry Number line', setting the `symmetry` variable no longer has an effect because the rotors are already defined.

Reply:
Thanks for the observations!
If Symmetry in `None`, I think that `symfromlog` is being set to `True` in statmech.py (https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/rmgpy/cantherm/statmech.py#L251)
The other part you pointed out seems problematic, I'll address it in a fixup commit

Reply:
Gotcha, good catch! I should have checked statmech.py